akka-cluster-sharding/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -3,3 +3,9 @@ ProblemFilters.exclude[Problem]("akka.cluster.sharding.Shard.*")
 
 # #25191
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion.retryTask")
+
+# #27299 Separate cluster sharding readiness from stats query
```

Comment:
Could you add a comment why these are safe? The first two seem to be public API but `@DoNotInherit`, the last two are private implementation methods.